index.js

```diff
@@ -6,4 +6,6 @@ import ActionSheetAndroid from './ActionSheetAndroid';
 
 var ActionSheet = Platform.OS === 'ios' ? ActionSheetIOS : ActionSheetAndroid;
 
-export default ActionSheet;
+//export default ActionSheet;
+
+module.exports = ActionSheet;
```

Comment:
This is ES5 equivalent of what I've got in the past. Can you revert this line back to ES6 style? I'll then merge this PR. Thank you.